CORRELATION_CONTEXT_HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,64 @@
+# Correlation Context HTTP Header Format
+
+A correlation context header is used to pass the name-value context properties (also known as tags) for the trace. This is a companion header for the `Trace-Context`. The values are intended for control of logging systems and should be passed along to any child requests. 
+
+# Format
+
+## Header name
+
+`Correlation-Context`
+
+Multiple correlation context headers are allowed. Values can be combined in a single header according to [rfc](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2).
+
+## Field value
+
+`name1=value1[;type=(number|bool|string)], name2=value2[;type=(number|bool|string)]`
+
+Overall Correlation-Context length MUST NOT exceed `1024` bytes. Key and value length should stay well under the combined limit of `1024` bytes. 
+
+## Supported value types
+
+### String
+
+Default value type. Represents a url encoded string value
+
+**Examples**:
+
+```
+component=Frontend
+component=Frontend;type=string
+component=Front%3Dend
+```
+
+### Boolean
+
+Binary flag. Supported values `true` and `false`
```

Comment:
If we talk about save space, easy to save here by using 1 character to represent this.

Reply:
like it